### ISSUE TITLE
Question about cuda version

### ISSUE THREAD

**Issue by nemo0526** [coder-wangzhen/AIC22-MCVT#2]
Greetings! I wanna ask a question that pytorch1.10.1 can only support cuda 10.2 and 11.3 on the offcial website , will there be an error for it ?

**Comment by coder-wangzhen** [comment]
Hi, @nemo0526 ,

We didn't encounter version incompatibility problem. The basic environments such as pytorch and cuda we use are based on nvidia's pytorch image. The pytorch version inside is not a release version, but a github commit version (https://github.com/pytorch/pytorch/commit/3fd9dcf). You can see the specific base environment at the this link: https://docs.nvidia.com/deeplearning/frameworks/pytorch-release-notes/rel_21-09.html#rel_21-09

Kind regards

**Comment by nemo0526** [comment]
Thanks for your comment!! However, I got an error when establishing the container you offer . It says " The NVIDIA Driver was not detected.  GPU functionality will not be available.",but my driver on nvidia-smi is well to be detected , have you met this kind of problem?

![driver](https://user-images.githubusercontent.com/63770299/180658197-47530cfe-5364-4e79-879b-00fa168d2c46.png)


![smi](https://user-images.githubusercontent.com/63770299/180658088-91a7b260-e0c7-4ec7-8e10-247746066333.jpg)

**Comment by coder-wangzhen** [comment]
Hi, we haven't met this problem. Are you using WSL in windows? You can use the following code to verify whether CUDA is available.
```
import torch
print(torch.version.cuda)
print(torch.cuda.is_available())
```

**Comment by nemo0526** [comment]
Yes, I am using WSL2 on win10 . I'm so sorry that the code you give can detect the cuda ,so the problem isn't solved. 
By the way, will the image install the driver and cuda automatically?
![torch](https://user-images.githubusercontent.com/63770299/180737414-bad1c512-a312-455a-a0ca-560be21b1234.png)

**Comment by coder-wangzhen** [comment]
It should be OK that driver version >= 470.57.02. I didn't make it clear in my last reply. You should run this code in the docker container, and if no errors are reported, I think it should be possible to run our program in this container.

But, we haven't tested our container and program on WSL2. The reason for this WARNING may be that NVIDIA's container mistakenly thinks that there is no NVIDIA driver in WSL2 when it starts to check, but the driver and CUDA is actually available.

> ![driver](https://user-images.githubusercontent.com/63770299/180658197-47530cfe-5364-4e79-879b-00fa168d2c46.png)

**Comment by nemo0526** [comment]
It can detect the cuda lol.I am now trainig by cpu temporary,and I will figure out how to change to gpu. Thanks for your comment!

![inside container](https://user-images.githubusercontent.com/63770299/180792450-561d4dd5-21da-4b22-9368-129a43fc3ec0.png)
